jupyter_book/page.py

```diff
@@ -35,8 +37,26 @@ def build_page(path_ntbk, path_html_output, path_media_output=None, execute=Fals
 
     ########################################
     # Load in the notebook
-    notebook_name = op.splitext(op.basename(path_ntbk))[0]
-    ntbk = jpt.read(path_ntbk)
+    notebook_name, suff = op.splitext(op.basename(path_ntbk))
+
+    if suff in ['.md', '.markdown']:
+        # If it's a markdown file, we need to check whether it's a jupytext format
+        with open(path_ntbk, 'r') as ff:
+            lines = ff.readlines()
+            yaml, content = _split_yaml(lines)
+            yaml = YAML().load(''.join(yaml))
+
+        if (yaml is not None) and yaml.get('jupyter', {}).get('jupytext'):
+            # If we have jupytext metadata, then use it to read the markdown file
+            ntbk = jpt.reads(''.join(lines), 'md')
+        else:
+            # Otherwise, create an empty notebook and add all of the file contents as a markdown file
+            ntbk = nbf.v4.new_notebook()
+            ntbk['cells'].append(nbf.v4.new_markdown_cell(source=''.join(content)))
```

Comment:
Hi Chris, yes sure, at least this way we're going to fix the rendering of the Jupyter Book book itself. I think this is good enough for a quick fix, and as you suggest we will follow-up on this.